File: C-plus-plus_questions/classes_and_Objects/question2.cpp
#include <iostream>
#include <string>
using namespace std;

class Student
{
	public:
		std::string name;
		int roll_no;
		int phone;
		std::string address;
};

int main(){
	// sam details
	Student sam;
	sam.address = "ad124554";
	sam.name = "Sam";
	sam.phone = 0711111111;
	sam.roll_no = 2;

	cout << "Address: " << sam.address << "\n";
	cout << "phone: " << sam.phone << "\n";
	cout << "Name: " << sam.name << "\n";
	cout << "roll_no: " << sam.roll_no << "\n";

	//John details
	Student John;
	John.address = "ad174hfas";
	John.name = "John";
	John.phone = 072222222;
	John.roll_no = 3;

	cout << "\nAddress: " << John.address << "\n";
	cout << "phone: " << John.phone << "\n";
	cout << "Name: " << John.name << "\n";
	cout << "roll_no: " << John.roll_no << "\n";
	
}

File: Python_questions/classes_and_Objects/question2.py
class Student:

    def __init__(self, name, roll_no, phone_no, address) -> None:
        self.name = name
        self.roll_no = roll_no
        self.phone_no = phone_no
        self.address = address


stud1 = Student("john", 2, "+254 099 4213", "Eldoret")
stud2 = Student("Sam", 4, "+254 583 7490", "Nairobi")
print(stud1.name, "phone no: ", stud1.phone_no)
print('\t\t', "roll no: ", stud1.roll_no)
print('\t\t', "address", stud1.address)

print(stud2.name, "phone no: ", stud2.phone_no)
print('\t\t', "roll no: ", stud2.roll_no)
print('\t\t', "address", stud2.address)
